libs/langchain-community/src/graphs/graph_document.ts

```diff
@@ -0,0 +1,86 @@
+import { Serializable } from "@langchain/core/load/serializable";
+import { Document } from "@langchain/core/documents";
+
+export class Node extends Serializable {
+  id: string | number;
+
+  type: string;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  properties: Record<string, any>;
+
+  lc_namespace = ["langchain", "graph", "document_node"];
+
+  constructor({
+    id,
+    type = "Node",
+    properties = {},
+  }: {
+    id: string | number;
+    type: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    properties?: Record<string, any>;
+  }) {
+    super();
```

Comment:
Should technically pass in all inputs `fields` here to truly make it serializable but no worries

Reply:
Will improve this in next PR that extends and uses this class.